pkg/scalers/rabbitmq_scaler.go

```diff
@@ -364,9 +364,9 @@ func (s *rabbitMQScaler) IsActive(ctx context.Context) (bool, error) {
 	}
 
 	if s.metadata.mode == rabbitModeQueueLength {
-		return float64(messages) > s.metadata.minMetricValue, nil
+		return messages > s.metadata.actionvationValue, nil
 	}
-	return publishRate > s.metadata.minMetricValue || float64(messages) > s.metadata.minMetricValue, nil
+	return publishRate > float64(s.metadata.actionvationValue) || float64(messages) > float64(s.metadata.actionvationValue), nil
```

Comment:
I get your point, makes sense. But IMO will be less clear in cases like Eg:  `int64(0.5) > 0` I guess in this case it should be **Active** since the publishRate is greater than default value but it will evaluate false. 

This is the final code.
```
return publishRate > float64(s.metadata.actionvationValue) || messages > s.metadata.actionvationValue, nil
```

Reply:
You're right, and thinking on it, that maintains current behaviour